bin/scrape_software_versions.py

```diff
@@ -4,48 +4,49 @@
-    'StringTie': ['v_stringtie.txt', r"(\S+)"],
-    'Preseq': ['v_preseq.txt', r"Version: (\S+)"],
-    'RSeQC': ['v_rseqc.txt', r"read_duplication.py ([\d\.]+)"],
-    'Qualimap': ['v_qualimap.txt', r"QualiMap v(\S+)"],
-    'dupRadar': ['v_dupRadar.txt', r"(\S+)"],
-    'edgeR': ['v_edgeR.txt', r"(\S+)"],
-    'MultiQC': ['v_multiqc.txt', r"multiqc, version (\S+)"],
+    "nf-core/rnaseq": ["v_ngi_rnaseq.txt", r"(\S+)"],
+    "Nextflow": ["v_nextflow.txt", r"(\S+)"],
+    "FastQC": ["v_fastqc.txt", r"FastQC v(\S+)"],
+    "Cutadapt": ["v_cutadapt.txt", r"(\S+)"],
+    "Trim Galore!": ["v_trim_galore.txt", r"version (\S+)"],
+    "SortMeRNA": ["v_sortmerna.txt", r"SortMeRNA version (\S+),"],
+    "STAR": ["v_star.txt", r"(\S+)"],
+    "HISAT2": ["v_hisat2.txt", r"version (\S+)"],
+    "Picard MarkDuplicates": ["v_markduplicates.txt", r"([\d\.]+)-SNAPSHOT"],
+    "Samtools": ["v_samtools.txt", r"samtools (\S+)"],
+    "featureCounts": ["v_featurecounts.txt", r"featureCounts v(\S+)"],
+    "RSEM": ["v_rsem.txt", r"RSEM v(\S+)"],
+    "Salmon": ["v_salmon.txt", r"salmon (\S+)"],
+    "deepTools": ["v_deeptools.txt", r"bamCoverage (\S+)"],
+    "StringTie": ["v_stringtie.txt", r"(\S+)"],
+    "Preseq": ["v_preseq.txt", r"Version: (\S+)"],
+    "RSeQC": ["v_rseqc.txt", r"read_duplication.py ([\d\.]+)"],
+    "Qualimap": ["v_qualimap.txt", r"QualiMap v(\S+)"],
+    "dupRadar": ["v_dupRadar.txt", r"(\S+)"],
+    "edgeR": ["v_edgeR.txt", r"(\S+)"],
+    "MultiQC": ["v_multiqc.txt", r"multiqc, version (\S+)"],
 }
 results = OrderedDict()
-results['nf-core/rnaseq'] = '<span style="color:#999999;\">N/A</span>'
-results['Nextflow'] = '<span style="color:#999999;\">N/A</span>'
-results['FastQC'] = '<span style="color:#999999;\">N/A</span>'
-results['Cutadapt'] = '<span style="color:#999999;\">N/A</span>'
-results['Trim Galore!'] = '<span style="color:#999999;\">N/A</span>'
-results['SortMeRNA'] = '<span style="color:#999999;\">N/A</span>'
-results['STAR'] = False
-results['HISAT2'] = False
-results['Picard MarkDuplicates'] = '<span style="color:#999999;\">N/A</span>'
-results['Samtools'] = '<span style="color:#999999;\">N/A</span>'
-results['featureCounts'] = '<span style="color:#999999;\">N/A</span>'
-results['Salmon'] = '<span style="color:#999999;\">N/A</span>'
-results['StringTie'] = '<span style="color:#999999;\">N/A</span>'
-results['Preseq'] = '<span style="color:#999999;\">N/A</span>'
-results['deepTools'] = '<span style="color:#999999;\">N/A</span>'
-results['RSeQC'] = '<span style="color:#999999;\">N/A</span>'
-results['dupRadar'] = '<span style="color:#999999;\">N/A</span>'
-results['edgeR'] = '<span style="color:#999999;\">N/A</span>'
-results['Qualimap'] = '<span style="color:#999999;\">N/A</span>'
-results['MultiQC'] = '<span style="color:#999999;\">N/A</span>'
+results["nf-core/rnaseq"] = '<span style="color:#999999;">N/A</span>'
+results["Nextflow"] = '<span style="color:#999999;">N/A</span>'
```

Comment:
@grst , I'm not sure that these editor-config related changes are necessary. Or perhaps we are adapting this as a convention throughout?

Reply:
We don't usually enforce this kind of thing (except for maybe tabs/spaces). I wrote this code originally I think and I prefer single quotes, everyone has their own preference 😉 Generally I think it's nicest to just stick with whatever is already there though, saves a massive diff and means the line history is fair.

Reply:
Though here it looks like the backslashes need removing in the span, so it doesn't really matter.

Reply:
Seems like I didn't pay attention to my [black](https://github.com/psf/black). 
I agree that diffs should be minimal and will revert this (except for the superfluous backslashes I guess)